tests/stats_tracker_test.cpp

```diff
@@ -875,6 +875,7 @@ struct test_subscriber : public event_subscriber {
 
 TEST_CASE( "achievements_tracker_in_game", "[stats]" )
 {
+    get_stats.clear();
```

Comment:
```suggestion
    get_stats().clear();
```
I'm guessing you meant this?

Reply:
agggh